pkgs/development/python-modules/rauth/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+, requests
+, pytestCheckHook
+, mock
+, nose
+, pycrypto
+}:
+
+buildPythonPackage rec {
+  pname = "rauth";
+  version = "0.7.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "litl";
+    repo = "rauth";
+    rev = version;
+    hash = "sha256-wRKZbxZCEfihOaJM8sk8438LE++KJWxdOGImpL1gHa4=";
+  };
+
+  patches = [
+    (fetchpatch {
+      name = "fix-pycrypdodome-replacement-for-pycrypto.patch";
```

Comment:
Please add a comment pointing to https://github.com/litl/rauth/pull/211

Reply:
Addressed in [force-push](https://github.com/NixOS/nixpkgs/compare/92c842ad18ddbf42b1a939f87e57beb1a8925094..57e49fa1057264791d33ba6db9b20aeb356be259)